app/models/router/responses/GetGoodsRecordResponse.scala

```diff
@@ -16,58 +16,94 @@
 
 package models.router.responses
 
-import play.api.libs.json.{OWrites, Reads, __}
+import play.api.libs.json.{JsSuccess, JsValue, Json, Reads, Writes}
 
 import java.time.Instant
 
-final case class GetGoodsRecordResponse(
+case class GetGoodsRecordResponse(
   recordId: String,
-  commodityCode: String,
-  countryOfOrigin: String,
+  eori: String,
+  actorId: String,
   traderRef: String,
+  comcode: String,
+  adviceStatus: String,
   goodsDescription: String,
+  countryOfOrigin: String,
+  category: Int,
+  assessments: Option[Seq[Assessment]] = None,
+  supplementaryUnit: Option[Double] = None,
+  measurementUnit: Option[String],
+  comcodeEffectiveFromDate: Instant,
+  comcodeEffectiveToDate: Option[Instant] = None,
+  version: Int,
+  active: Boolean,
+  toReview: Boolean,
+  reviewReason: Option[String] = None,
   declarable: String,
+  ukimsNumber: Option[String] = None,
```

Comment:
think ukimsNumber is still mandatory at the moment

Reply:
I copied from the router model which has it as optional

https://github.com/hmrc/trader-goods-profiles-router/blob/main/app/uk/gov/hmrc/tradergoodsprofilesrouter/models/response/CreateOrUpdateRecordResponse.scala#L46